content/docs/components/serving/tfbatchpredict.md

```diff
@@ -1,9 +1,15 @@
 +++
 title = "TensorFlow Batch Predict"
-description = "Batch prediction for TensorFlow models"
+description = "See Kubeflow [v0.6 docs](https://v0-6.kubeflow.org/docs/components/serving/tfbatchpredict/) for Batch Prediction with TensorFlow models"
```

Comment:
Use lower case for "batch prediction", because batch prediction isn't a product name.

Reply:
Done!